.packit.yml

```diff
@@ -72,6 +72,9 @@ jobs:
     trigger: commit
     owner: '@centos-automotive-sig'
     project: bluechi-snapshot
+    notifications:
+      failure_comment:
```

Comment:
If I understand the `trigger: commit` correctly, this comment will be added each time the build(s) fail - also for not yet merged PRs, right? I think it would be great to have the comment only when the PR is not yet merged. 
I'd suggest to duplicate the `copr_build` job - one without the notification and one with. To filter based on merged to main then add `branch: '(?!main)'` to the former and `branch: main` to the latter (with the notification). 
What do you think?

Reply:
If the PR is not merged, then you can see errors *Checks* section at the bottom of the PR, so it doesn't seem to me worth to add additional comment about error.

But when PR is merged (that's `trigger:commit` meaning), then the only way to find a failure is to take a look at https://copr.fedorainfracloud.org/coprs/g/centos-automotive-sig/bluechi-snapshot/builds/ manually and check. So adding a comment to PR about failure after merge would make the error much more visible.

Reply:
Based on 
https://packit.dev/docs/configuration/upstream/upstream_koji_build#supported-triggers
`trigger: commit` will lead to the job being run on each commit for the specific branch. So if no branch is specified, its basically for all. 

> If the PR is not merged, then you can see errors Checks section at the bottom of the PR, so it doesn't seem to me worth to add additional comment about error.

I agree. Thats why I would distinguish the commit based on branch - commit to main means a PR is merged.